students/chiyoung/users/views.py

```diff
@@ -41,4 +41,27 @@ def post(self, request):
             return JsonResponse({"MESSAGE": "KEY_ERROR"}, status=400)
 
         except ValidationError as verr :
-            return JsonResponse({"MESSAGE": verr.message}, status=400)
\ No newline at end of file
+            return JsonResponse({"MESSAGE": verr.message}, status=400)
+
+class LoginView(View):
+    def post(self, request):
+        
+        try :
+            login_data = json.loads(request.body)
+
+            email    = login_data["email"]
+            password = login_data["password"]
+
+            if not Accounts.objects.filter(email=email, password=password).exists() :
+                raise ValidationError("INVALID_USER")
+
+            return JsonResponse({"MESSAGE": "SUCCESS"}, status=201)
+
+        except json.decoder.JSONDecodeError :
+            return JsonResponse({"MESSAGE": "REQUEST_WITHOUT_DATA"}, status=400)
```

Comment:
에러 핸들링 잘 해주셨습니다-! json decode error는 데이터가 없다기 보다는 json 형식이 아닌 경우에 발생하는 문제입니다.

"REQUEST_WITHOUT_DATA" 보다는 `"JSON_DECODE_ERROR"`가 더 정확할 것 같습니다.

```suggestion
            return JsonResponse({"MESSAGE": "REQUEST_WITHOUT_DATA"}, status=400)
```